content/news/xxxx-xx-xx-mixxx-accepted-for-gsoc-2023.md

```diff
@@ -0,0 +1,15 @@
+title: Mixxx accepted for GSoC 2023!
+authors: Daniel Schürmann
+tags: gsoc, gsoc-2023
+status: draft
+comments: no
```

Comment:
```suggestion
comments: yes
```
You forgot to re-enable the comments, or do you insist to keep them disabled?

Reply:
I want the comments disabled, to not have to deal with students applications in the comments.

Reply:
May I suggest to delete the last section of the article, than the last lines will be how to reach us via the public Zulip chat.
IMHO it's Ok, to disable the comments, if we mention another public communication channel at the end of the article.

Reply:
The next two blog entries describe the work of the last students anyway.

Reply:
> I want the comments disabled, to not have to deal with students applications in the comments.

We simply ignore those, we have clearly stated in the article that we only communicate with them through Zulip or GSoC. If they post applications in the comments, they have simply not read the blogpost and we can ignore those application comments. 

> IMHO it's Ok, to disable the comments, if we mention another public communication channel at the end of the article.

That was not the point. Comments are a way for the community to voice feedback on articles, disabling that for the sole reason that we want to avoid the spam from students that can't be bothered to read the article they're commenting on is unjustified.